Automattic/wp-calypso
# Add deprecated-editor-group flag to existing call to get editor details

#### Changes proposed in this Pull Request

* Adds a flag to indicate that the user has been added to the deprecated editor group

Related diff D44256-code

#### Testing instructions

* Run this PR locally logging in with user that doesn't match Diff D44256's requirements to be in the deprecation group
* Edit a post
* Inspect the Redux state and confirm that under `currentUser` there is a property `inEditorDeprecationGroup` set to false.
* Login with a user that does match D44256's requirements to be in the deprecation group or sandbox API and fudge that value in the API response to be truthy
* Switch back to editing a post
* Confirm the state `currentUser.inEditorDeprecationGroup` value is now set to true.
